script.js

```diff
@@ -187,6 +189,47 @@ function prb_plus()
   var x = document.getElementById('prb_count').innerHTML.split(": ");
   var cnt=Number(x[x.length-1])+1;
   document.getElementById('prb_count').innerHTML="Problems Count: "+cnt.toString();
+  updateAverage(cnt);        
+}
+
+//updating average when plus or minus  button is pressed
+function updateAverage(cnt){
+  var minutes='00',seconds='00',time='';
+  if(cnt==0){
+    runningAverage=0;
+    document.getElementById('Average time').innerHTML="Average time/per problem(in minutes): 0";
+  }
+  else{
+    timeNow = new Date().getTime();
+    difference =timeNow-T[2].timerStarted;  //Updating the runningAverage using "current_timestamp - previous timestamp(T[2].timerStarted)"
```

Comment:
Please change `timerstarted` variable name to something else, since we have already used this name, in this file